ftl/core/browsing.ftl

```diff
@@ -100,6 +101,7 @@ browsing-suspended = Suspended
 browsing-tag-duplicates = Tag Duplicates
 browsing-tag-rename-warning-empty = You can't rename a tag that has no notes.
 browsing-target-field = Target field:
+browsing-toggle-cards-notes-mode = Toggle Cards/Notes Mode
```

Comment:
It's fine as-is, but maybe "Toggle Showing Cards/Notes" would be slightly clearer?